cmake/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(cmake)
+
+set(CMAKE_CXX_STANDART 14)
```

Comment:
Why have you set the standard? You are not using any special things that is allowed only after c++14

Reply:
+